Sources/Build/BuildPlan.swift

```diff
@@ -831,7 +831,7 @@ public final class SwiftTargetBuildDescription {
         stream <<< "  },\n"
 
         // Write out the entries for each source file.
-        let sources = target.sources.paths
+        let sources = target.sources.paths + derivedSources.paths
         for (idx, source) in sources.enumerated() {
             let object = objects[idx]
```

Comment:
`objects` are formed by taking `sources + derivedSources` and adding a `.o` extension so this enumeration will still hold.